x-pack/plugins/beats_management/server/lib/adapters/database/kibana_database_adapter.ts

```diff
@@ -86,6 +87,21 @@ export class KibanaDatabaseAdapter implements DatabaseAdapter {
     return result;
   }
 
+  public async searchAll<Source>(
+    user: FrameworkUser,
+    params: DatabaseSearchParams
+  ): Promise<DatabaseSearchResponse<Source>> {
+    const result = await this.callWithUser(user, 'search', {
+      ...params,
+      scroll: '1m',
+      body: {
+        ...params.body,
+        size: 1000,
```

Comment:
One thing to keep in mind is exposing `size` as a configurable value so it can be tweaked by users if their use case is something we didn't expect. I'd recommend thinking about setting all these sizes as configurable, probably from the same config (as I think most of those are set to 1000)

Reply:
ah good catch!

As for translations... yea... 😊